Project-Service-implementation-Script/main.py

```diff
@@ -437,11 +437,8 @@ def programsFileCheck(filePathAddPgm, accessToken, parentFolder, MainFilePath):
                         # check if program is created or not 
                         if getProgramInfo(accessToken, parentFolder, extIdPGM):
                             print("Program Created SuccessFully.")
-                    else :
-                         userDetails = fetchUserDetails(environment, accessToken, dictDetailsEnv['projectService username/user id/email id/phone no. of Program Designer'])
-                         userId = userDetails[0]
-                         programCreation(accessToken,parentFolder,extIdPGM,programNameInp,proDesc,roles,userId)
-                         getProgramInfo(accessToken, parentFolder, extIdPGM)
+                        else :
```

Comment:
@MallanagoudaB any reason why we replaced this else condition?

Reply:
We suppose to call programCreate only once if the program is not available

Reply:
verified from old script sun bird script